generators/server/templates/src/main/java/package/config/CRLFLogConverter.java.ejs

```diff
@@ -14,6 +14,12 @@ import org.springframework.boot.ansi.AnsiElement;
 import org.springframework.boot.ansi.AnsiOutput;
 import org.springframework.boot.ansi.AnsiStyle;
 
+/**
+ * Log filter to prevent attackers from forging log entries by submitting input containing CRLF characters.
```

Comment:
```suggestion
 * Log filter to prevent attackers from forging log entries by submitting input containing CRLF characters.
 * CRLF characters are replaced with a red colored _ character.
```